LICENSE-binary

```diff
@@ -251,12 +258,25 @@ io.prometheus:simpleclient
 io.prometheus:simpleclient_common
 io.prometheus:simpleclient_dropwizard
 io.prometheus:simpleclient_servlet
+io.swagger.core.v3:swagger-annotations
+io.swagger.core.v3:swagger-annotations-jakarta
+io.swagger.core.v3:swagger-core
+io.swagger.core.v3:swagger-core-jakarta
+io.swagger.core.v3:swagger-integration
+io.swagger.core.v3:swagger-jaxrs2
+io.swagger.core.v3:swagger-models
+io.swagger.core.v3:swagger-models-jakarta
 org.apache.zookeeper:zookeeper
 
 BSD 3-Clause
 ------------
 org.scala-lang:scala-library
 
+Eclipse Distribution License - v 1.0
```

Comment:
Add the static content into `license-binary/` https://www.eclipse.org/org/documents/edl-v10.html